website/docs/docusaurus-core.md

```diff
@@ -113,6 +113,18 @@ function Home() {
 }
 ```
 
+### `<BrowserOnly/>`
+
+This component excludes its `children` during the prerendering stage of the build process. This is useful for hiding code that is only meant to run in the browsers (e.g. where the `window`/`document` objects are being accessed).
+
+```jsx {1,7}
```

Comment:
If we're highlighting the whole block I think it's better we don't highlight at all.